auth/src/main/java/software/amazon/awssdk/auth/signer/internal/AwsSignerExecutionAttribute.java

```diff
@@ -59,6 +59,16 @@
      */
     public static final ExecutionAttribute<Instant> PRESIGNER_EXPIRATION = new ExecutionAttribute<>("PresignerExpiration");
 
+    /**
+     * The key to specify whether to enable chunked encoding or not
+     */
+    public static final ExecutionAttribute<Boolean> ENABLE_CHUNKED_ENCODING = new ExecutionAttribute<>("ChunkedEncoding");
```

Comment:
I don't think these params are applicable to AwsSignerExecutionAttribute class. As of now, they are only applicable for S3 signer. 
Maybe we can  move them to a separate class or add ReviewBeforeRelease tag to finalize before GA.

Reply:
yeah, I'll move them to a separate class